iFixit/product-page-components
# Shopify: Fill in recommended Product schema jsonld fields for SEO

Injects the new jsonld we are building on the backend (see https://github.com/iFixit/ifixit/pull/40347) into the html `<head>`. We don't need to build the jsonld in JS anymore, so we can drop that code.

Also updates some testing data and a snapshot test.

## QA
Start the dev server and make sure the jsonld script tag gets added IFF the jsonld is not empty. Copy the `<head>` into [google's rich results tool](https://search.google.com/test/rich-results) and make sure there are no warnings or errors.

CC: @sterlinghirsh @JFleischhauer @GrantMatejka

Closes https://github.com/ifixit/ifixit/issues/39560
